### DISCUSSION TITLE
Is it possible to clear out all records leaving the structure intact?

### DISCUSSION THREAD

**Discussion by delliottg** [pocketbase/pocketbase#4799] [Q&A]
I'm taking a SaaS class that uses Pocketbase.  In my code I'm receiving a 404 error "Missing required record id".  I've gone back through the instructions for each of the API rules and while I think I've been diligent, it's entirely possible I'm missing or misconfigured something.  What I want to do is import another student's database structure, but I don't want their data.  Is it possible to clear the data from a given data.db file and leave behind the structure?

Another option might be the ability to diff the two files using something like Meld?   At least that way I might be able to figure out where the discrepancy is.

**Comment by ganigeorgiev** [comment]
I'm not sure I follow.

You can export/import a collections structure from the _Admin UI > Settings > Export/Import collections_.

If that's not what you are looking for, please elaborate a little more on what you are trying to do and what request you are sending for triggering the 404 error.

**Comment by delliottg** [comment]
Thanks for this answer, I’m going to try to export the database structure from my problematic database using your instructions.  Hopefully I can get a response from someone in the class on Discord so I can show them how to export theirs.  I don’t need their data, just the database structure (I think).
